tools/firely-server-import.rst

```diff
@@ -0,0 +1,182 @@
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| -l, --limit <limit>                       | limit                            |          | Limit the number of resources to import. Use this for testing your setup.                                                                  |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --license <license>                       | license                          | yes      | Firely Server license file                                                                                                                 |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --update-existing-resources <true|false>  | updateExistingResources          |          | When true, a resource is updated in the database if it already exists. Otherwise a resource is updated and history is created.             |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| -c, --connectionstring <connectionstring> | sqlServer/connectionString       | yes      | Connection string to Firely Server SQL Server database                                                                                     |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --sqlPar <sqlPar>                         | sqlServer/saveParallel           |          | The # of batches to save in parallel. Depends on your bandwidth to SQL Server and its processing power.                                    |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --sqlBatch <sqlBatch>                     | sqlServer/saveBatchSize          |          | The # of resources to save in each batch. SQL Server must be able to process it within the CommandTimeout.                                 |
```

Comment:
Should not be too small either. SQL BulkCopy is not optimally efficient with < 1000 records or so.

Reply:
we have 500 records per batch by default. Shall we change it then? 
I will make a remark about the recommended batch size of at least 500 records for now.